src/dpq2/conv/to_d_types.d

```diff
@@ -31,6 +31,7 @@ alias PGuuid =          UUID; /// UUID
 alias PGdate =          Date; /// Date (no time of day)
 alias PGtime_without_time_zone = TimeOfDay; /// Time of day (no date)
 alias PGtimestamp_without_time_zone = TimeStampWithoutTZ; /// Both date and time (no time zone)
+alias PGtimestamp_with_time_zone = SysTime; /// Both date and time with time zone
```

Comment:
No, sorry! `timestamp with time zone` isn't supported by binary protocol (as I know).
Assumtion what you can convert on the fly between `TimeStampWithoutTZ` and `timestamp with time zone` is wrong.

Maybe my knowledge about support of `timestamp with time zone` type is wrong too - it is necessary to investigate this question.

Also, as workaround, I can propose text interchange of  `timestamp with time zone` values.

Reply:
I think we've solved this in the comments up?

Reply:
Yes